Makefile

```diff
@@ -168,6 +168,30 @@ navigation-fixtures:
 	./gradlew :libnavigation-util:generateDependencyGraphMapboxLibraries
 	./gradlew :libnavigation-core:generateDependencyGraphMapboxLibraries
 
+.PHONY: 1.0-core-check-api
+1.0-core-check-api:
+	./gradlew :libdirections-offboard:checkApi
+	./gradlew :libdirections-hybrid:checkApi
+	./gradlew :libdirections-onboard:checkApi
+	./gradlew :libnavigation-base:checkApi -PhidePackage=com.mapbox.navigation.base.internal
```

Comment:
`base` module has one more internal package `com.mapbox.navigation.base.route.internal`

Reply:
Nah, not anymore. That package was removed in https://github.com/mapbox/mapbox-navigation-android/pull/3043